lib/eslint.js

```diff
@@ -798,7 +802,7 @@ module.exports = (function() {
                 impliedStrict: ecmaFeatures.impliedStrict,
                 ecmaVersion: ecmaVersion,
                 sourceType: currentConfig.parserOptions.sourceType || "script",
-                childVisitorKeys: espree.VisitorKeys,
+                childVisitorKeys: parser ? parser.VisitorKeys : espree.VisitorKeys,
                 fallback: "none"
```

Comment:
There's only one other place, correct? I'd handle that as a separate issue. It looks like it never would have worked completely, so let's track it as a bug. If there are other places, please list them in an issue.